cmake/modules/FindMaya.cmake

```diff
@@ -261,15 +260,6 @@ if(MAYA_INCLUDE_DIRS AND EXISTS "${MAYA_INCLUDE_DIR}/maya/MTypes.h")
     endif()
 endif()
 
-if(MAYA_INCLUDE_DIRS AND EXISTS "${MAYA_INCLUDE_DIR}/maya/MDefines.h")
-    file(STRINGS ${MAYA_INCLUDE_DIR}/maya/MDefines.h MAYA_PREVIEW_RELEASE_VERSION REGEX "#define MAYA_PREVIEW_RELEASE_VERSION.*$")
-    if(MAYA_PREVIEW_RELEASE_VERSION)
-        string(REGEX MATCHALL "[0-9]+" MAYA_PREVIEW_RELEASE_VERSION ${MAYA_PREVIEW_RELEASE_VERSION})
-    else()
-        set(MAYA_PREVIEW_RELEASE_VERSION 0)
-    endif()
-endif()
```

Comment:
This file will be removed from future versions of Maya, so remove the check for it. We currently aren't using any preview release checks.